bitnami/rabbitmq/values.yaml

```diff
@@ -188,7 +188,7 @@ memoryHighWatermark:
 
 ## @param plugins List of default plugins to enable (should only be altered to remove defaults; for additional plugins use `extraPlugins`)
 ##
-plugins: "rabbitmq_management rabbitmq_peer_discovery_k8s"
+plugins: "rabbitmq_management rabbitmq_peer_discovery_k8s rabbitmq_shovel rabbitmq_shovel_management"
```

Comment:
In the commit that changes this line, the commit-message says:
> … and disable rabbitmq_prometheus

But I don't see this change in practice, so there might be a small typo somewhere.

Also, do we really want to disable it ?
I'm thinking that we could keep it enabled by default, because (I'm guessing that) it has no big drawback (maybe security?) and it could be useful one day for a one-shot debug (get metrics that datadog doesn't provide, by doing a curl on the metrics endpoint manually).
WDYT ?